config/pipeline.yaml

```diff
@@ -921,7 +921,7 @@ jobs:
   rt-tests: &rt-tests
     template: rt-tests.jinja2
     kind: job
-    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-rt/20240313.0/{debarch}'
+    nfsroot: 'https://storage.kernelci.org/images/rootfs/debian/bookworm-rt/20240806.0/{debarch}'
```

Comment:
https://lava.collabora.dev/scheduler/job/15194120 ran on 16th August. It is using https://storage.chromeos.kernelci.org/images/rootfs/debian/bookworm-cros-flash/20240422.0/amd64/initrd.cpio.gz which is completely wrong. I'm not sure what's going on. I'll work with KernelCI team to debug it.

Reply:
@pawiecz @nuclearcat please have a look here

Reply:
Thats because -rt jobs defined in platforms-chromeos.yaml where they don't belong.
arm64-chromebook-device have in definition bookworm-cros-flash as all jobs here is chromeos jobs.

Reply:
@nuclearcat Both builds and tests for rt are defined in pipeline.yaml (a fix has been merged last or last to last week). What are you referring to in platforms-chromeos?